### ISSUE TITLE
[Accessibility] Toast notifications not announced by screen readers

### ISSUE THREAD

**Issue by patrickhlauke** [bitwarden/clients#12538]
### Steps To Reproduce

1. Run a screen reader
2. Perform an action that triggers a toast notification (e.g. copying a value from an entry)

### Expected Result

Toast notification should be announced by the screen reader (using a live region notification).

### Actual Result

Toast notification is not announced at all by screen readers.

### Screenshots or Videos

https://github.com/user-attachments/assets/d797c0b6-6296-46b9-8cc4-47e04d7441c2

Video: Windows/NVDA, copying a value from an entry - a toast notification is shown, but NVDA remains fully silent.

### Additional Context

Possibly still related to https://github.com/bitwarden/clients/issues/3487

I also filed an issue ages ago upstream https://github.com/scttcper/ngx-toastr/issues/957 but not sure if anything came from that.

This may be interesting/relevant: https://tetralogical.com/blog/2024/05/01/why-are-my-live-regions-not-working/

Matching browser/extension issue: https://github.com/bitwarden/clients/issues/12528

### Operating System

Windows

### Operating System Version

11

### Installation method

Direct Download (from bitwarden.com)

### Build Version

2024.12.1

### Issue Tracking Info

- [x] I understand that work is tracked outside of GitHub. A PR will be linked to this issue should one be opened to address it, but Bitwarden doesn't use fields like "assigned", "milestone", or "project" to track progress.

**Comment by SergeantConfused** [comment]
Hello @patrickhlauke,

Thank you for your report. I have flagged this to the Engineering department; please feel free to post additional information, such as screenshots or a screen video recordings, if you wish.

Thank you again,

**Comment by danielleflinn** [comment]
Closing this as duplicate of https://github.com/bitwarden/clients/issues/3487; the team will be refactoring this component soon to work across web clients. We'll use the issue linked above to track when the issue is resolved.
